src/go/rpk/pkg/cli/cmd/debug/bundle.go

```diff
@@ -76,15 +83,25 @@ func newBundleCommand(fs afero.Fs) *cobra.Command {
 			logsLimit, err := units.FromHumanSize(logsSizeLimit)
 			out.MaybeDie(err, "unable to parse --logs-size-limit: %v", err)
 
-			err = executeBundle(cmd.Context(), fs, cfg, cl, admin, logsSince, logsUntil, int(logsLimit), timeout)
+			err = executeBundle(cmd.Context(), fs, cfg, cl, admin, logsSince, logsUntil, int(logsLimit), timeout, path)
 			out.MaybeDie(err, "unable to create bundle: %v", err)
 		},
 	}
 	command.Flags().StringVar(
 		&adminURL,
-		"admin-url",
+		config.FlagAdminHosts2,
+		"",
+		"Comma-separated list of admin API addresses (<IP>:<port>)",
+	)
+	command.Flags().StringVar(&adminURL, "admin-url", "", "")
+	command.Flags().MarkDeprecated("admin-url", "use --"+config.FlagAdminHosts2)
+
+	command.Flags().StringVarP(
+		&outFile,
+		outputFlag,
+		"o",
 		"",
-		"The address to the broker's admin API. Defaults to the one in the config file",
+		"The file path where the debug file will be written (default to ./<timestamp>-bundle.zip)",
```

Comment:
`default ./<timesta...`, rather than `default to` -- per consistency with other flags